### PR TITLE
Updated SKY UX icons reference to 3.0.0-beta.1

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-skyux-icons-beta-1](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-skyux-icons-beta-1)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-skyux-icons-beta-1](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-skyux-icons-beta-1)

File: src/app/public/style-loader.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export class SkyAppStyleLoader {
         // when the font is loaded unless a known character with a different width
         // than the default is not specified.
         fontAwesome.load('\uf0fc', SkyAppStyleLoader.LOAD_TIMEOUT),
-        skyuxIcons.load('\ue808', SkyAppStyleLoader.LOAD_TIMEOUT),
+        skyuxIcons.load('\ue815', SkyAppStyleLoader.LOAD_TIMEOUT),
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/style-loader.ts:27]: You’ll need to update the node-utils loader as well.

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-skyux-icons-beta-1](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-skyux-icons-beta-1)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-skyux-icons-beta-1](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-skyux-icons-beta-1)

**Pull request merged**